tools/cluster-init.in

```diff
@@ -579,12 +563,6 @@ for host in $host_list; do
 	scp /etc/hosts root@${host}:/etc/hosts
     fi
 
-    if [ $pcmk_ver = 10 ]; then
-	scp /etc/hosts root@${host}:/etc/hosts
-	scp ~/.ssh/id_dsa.suse root@${host}:.ssh/id_dsa
-	scp ~/.ssh/known_hosts root@${host}:.ssh/known_hosts
-    fi
-
     ssh -l root ${host} -- grep -q "token:" $cs_conf 2>&1 > /dev/null
     new_config=$?
     new_config=1
```

Comment:
Typo in commit message - installed -> install

Reply:
Thanks, fixed